### PR TITLE
[4.15] Use timeout_sampler module

### PR DESCRIPTION
**Pull request by jpeimer**
Use https://github.com/RedHatQE/timeout-sampler for `TimeoutSampler`

### CODE REVIEW

**Comment by redhat-qe-bot1** [comment]
Report bugs in [Issues](https://github.com/myakove/github-webhook-server/issues)

The following are automatically added:
 * Add reviewers from OWNER file (in the root of the repository) under reviewers section.
 * Set PR size label.
 * New issue is created for the PR. (Closed when PR is merged/closed)
 * Run [pre-commit](https://pre-commit.ci/) if `.pre-commit-config.yaml` exists in the repo.

Available user actions:
 * To mark PR as WIP comment `/wip` to the PR, To remove it from the PR comment `/wip cancel` to the PR.
 * To block merging of PR comment `/hold`, To un-block merging of PR comment `/hold cancel`.
 * To mark PR as verified comment `/verified` to the PR, to un-verify comment `/verified cancel` to the PR.
        verified label removed on each new commit push.
 * To cherry pick a merged PR comment `/cherry-pick <target branch to cherry-pick to>` in the PR.
    * Multiple target branches can be cherry-picked, separated by spaces. (`/cherry-pick branch1 branch2`)
    * Cherry-pick will be started when PR is merged
 * To build and push container image command `/build-and-push-container` in the PR (tag will be the PR number).
 * To add a label by comment use `/<label name>`, to remove, use `/<label name> cancel`

<details>
<summary>Supported /retest check runs</summary>

 * `/retest tox`: Retest tox
 * `/retest python-module-install`: Retest python-module-install

</details>

<details>
<summary>Supported labels</summary>

 * hold
 * verified
 * wip
 * lgtm

</details>

File: ocp_resources/utils.py [hunk 1]
```diff
@@ -1,229 +1,7 @@
-import datetime
-import time
-from warnings import warn
-
-
 import yaml
 from simple_logger.logger import get_logger
 
 LOGGER = get_logger(name=__name__)
-DEPRECATED_MSG = (
```
Comment (myakove) [inline ocp_resources/utils.py:10]: Please do not remove the timeout sampler functionality from 4.15 branch.

Leave it as is, we may have samo users using it

**Review by myakove — CHANGES_REQUESTED** [review]

**Review by myakove — APPROVED** [review]

File: ocp_resources/utils.py (hunk 1 above)
Comment (jpeimer) [reply ocp_resources/utils.py]: Sure, thanks, left it as is

**Pull request merged**